src/Explorer/Tabs/NotebookV2Tab.ts

```diff
@@ -84,11 +85,13 @@ export default class NotebookTabV2 extends NotebookTabBase {
 
   protected getTabsButtons(): CommandButtonComponentProps[] {
     const availableKernels = NotebookTabV2.clientManager.getAvailableKernelSpecs();
+    const isNotebookUntrusted = this.notebookComponentAdapter.isNotebookUntrusted();
+
+    const runBtnTooltip = isNotebookUntrusted ? NotebookUtil.UntrustedNotebookRunHint : undefined;
```

Comment:
>UntrustedNotebookRunHint [](http://example.com/codeflow?start=61&length=24)

dont see this work in the preview url

Reply:
You can hover over the play buttons and see this tooltip.

https://user-images.githubusercontent.com/693092/128913241-b341e0f9-f0ea-41a8-9ab2-9b8a500d685b.mp4